server/implementation/src/main/java/io/smallrye/graphql/execution/ExecutionService.java

```diff
@@ -74,15 +74,20 @@ public ExecutionResponse execute(JsonObject jsonInput) {
 
         // ExecutionId
         ExecutionId finalExecutionId = ExecutionId.from(executionIdPrefix + executionId.getAndIncrement());
+        LogPayloadOption payloadOption = LogPayloadOption.OFF;
```

Comment:
Here - should we not get LogPayloadOption from config ? Setting to OFF will also not print anything ?